docs/fern/pages/reference/schemas/annotation-style.md

```
@@ -0,0 +1,47 @@
+---
+title: "Annotation style"
+description: "Visual overrides for this annotation. Anything unset falls back to the resolved `annotationDefaults` theme (test, then spec, then config), then to the built-in theme."
+---
+
+{/* Generated by docs/.scripts/buildSchemaReferencesV4.js. Do not edit by hand;
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)